README/DEV_DEPLOY.md

```diff
@@ -1,34 +1,38 @@
-## Hypothetical Deploying from Dev
+## Deploying from Dev
 
 ### Background
 
-As of 31 May 2024, SocialPredict does not have a production version launched. That is to say, SocialPredict is designed for development, meaning out of the box, everything it comes with can be worked on within a local machine, but it's not set up to be served securely as a production instance on the web that can't be easily hacked.
+As of 28 July 2024, SocialPredict does not have a production version launched. SocialPredict is designed for development: users can deploy software out of the box. Everything it comes with can be worked on within a local machine, but it's not set up to be served securely as a production instance on the web that can't be easily hacked.
 
 That being said, while not recommended, it is possible to adapt the development version of SocialPredict to run in a production environment in its current state, for small and experimental deployments.
 
-This document is geared at helping those who might want to convert the development version of socialpredict into a web deployment. While not all-encompassing and user caution is encouraged, this document can serve as a general guideline to get going.
+This document is geared at helping those who might want to convert the development version of SocialPredict into a web deployment. This document is not all-encompassing and users should exercise caution, but it is a set of guidelines to start deploying SocialPredict on the web.
```

Comment:
> This document is geared at helping those who might want to convert the development version of SocialPredict into a web deployment.

Suggest: This document is geared toward setting up what we have provided thus far, which is really arguably a dev deployment, and serving it as a prod deployment for experimental purposes.

Reply:
thanks, editing it!